smpe/bld/SMPMCS.txt

```diff
@@ -29,12 +29,41 @@
 ++SAMP(ZWE6DDEF)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(1) .
 ++SAMP(ZWE7APLY)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(1) .
 ++SAMP(ZWE8ACPT)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(1) .
+++SAMP(ZWECHG)       SYSLIB(SZWEEXEC) DISTLIB(AZWESAMP) RELFILE(2) .
+++SAMP(ZWEGEN00)     SYSLIB(SZWEEXEC) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWEMCOPY)     SYSLIB(SZWEEXEC) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWEOCOPY)     SYSLIB(SZWEEXEC) DISTLIB(AZWESAMP) RELFILE(2) .
```

Comment:
### `ZWEMCOPY`
* Here and in shell function [data_set_copy_to_data_set](https://github.com/zowe/zowe-install-packaging/blob/8334e19fa162a68303a83a44e0931bed1e08ef7a/bin/libs/zos-dataset.sh#L101), which is not used

### `ZWEOCOPY`
* Only here
* Could not find any usage of this

Time for small clean-up?